geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/GenerateAvroSchemaCommand.scala

```diff
@@ -20,11 +20,12 @@ class GenerateAvroSchemaCommand extends Command {
 
   override def execute(): Unit = {
     val sft = SftArgResolver.getSft(params.spec, params.featureName)
-    if (sft.isDefined) {
-      val schema = AvroSimpleFeatureUtils.generateSchema(sft.get, withUserData = true)
-      Command.output.info(schema.toString(true))
-    } else {
-      Command.user.warn(s"Could not find feature type ${params.spec}")
+    sft match {
```

Comment:
Actually now that I look at this, I'm not sure why this wasn't routed through the CLArgResolver. I've swapped it to that.